Making Sense Project API/Logic/ReadWriteJsonCar.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Making_Sense_Project.Model;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Making_Sense_Project.Logic
+{
+    public class ReadWriteJsonCar
+    {
+        private string _pathJsonCar;
```

Comment:
Tal vez no sea parte del alcance de este PR, pero para tenerlo en cuenta. Una vez asignado el valor de _pathJsonCar no debería cambiar, correcto? En caso de ser así podríamos declararlo como `readonly`:
```suggestion
        private readonly string _pathJsonCar;
```